schemas/README.md

```diff
@@ -0,0 +1,6 @@
+# Updating schema
```

Comment:
I'm not sure this file is necessary, instead just populate the main README

Reply:
Done.